### DISCUSSION TITLE
Ekuiper server in Docker image

### DISCUSSION THREAD

**Discussion by lupe120** [lf-edge/ekuiper#2408] [Q&A]
Which file in the Ekuiper docker image is responsible for starting/running the server? I believe it to be bin/kuiper. How can i manually start  bin/kuiper?

**Comment by lupe120** [comment]
Ekuiper server will start when the container is brought up, but to verify that kuiperd is responsible for starting the server, I added a CMD command with 'bin/./kuiperd' which replaces the command used to start the ekuiper server in the ekuiper image. This produces the same results had I not added this CMD.

**Comment by ngjaying** [comment]
It is bin/kupierd.  bin/kuiper is the RPC client.
